components/icons/awesome-screen-icon.css

```diff
@@ -1,14 +1,29 @@
-awesome-screen-icon{
+awesome-screen-icon {
     cursor: pointer;
-    display: inline-block;
-    margin: 3.5em;
     text-align: center;
-    line-height: 1em;
+    width: 9em;
+    display: inline-flex;
+    margin: 2.5em;
 }
 
-awesome-screen-icon img{
-    width: 4em;
+awesome-screen-icon .contentWrapper{
+    height: 8em;
+    width: 8em;
+    text-align: -webkit-center;
```

Comment:
if you use -webkit, also do EVERY other browser, otherwise just stick to the standards.

btw, Think this should just be center.